packages/one-app-bundler/__tests__/bin/serve-module.spec.js

```diff
@@ -34,15 +35,26 @@ const setup = (modulePath) => {
 };
 
 describe('serve-module', () => {
+  let originalNodeEnv;
+
   process.cwd = () => '/mocked';
   const originalPlatform = process.platform;
 
   const setPlatform = (platform) => Object.defineProperty(process, 'platform', {
     value: platform,
   });
 
+  beforeAll(() => {
+    originalNodeEnv = process.env.NODE_ENV;
+  });
+
   beforeEach(() => {
     setup('../my-module-name');
+    originalNodeEnv = 'production';
```

Comment:
I removed all the code related to this throughout the tests. Was initially trying to avoid a leak with global variables which would cause other tests NODE_ENV to change, but all looks good without it. Probably self inflicted at some point before.